android/src/main/java/com/pylon/RNSpokestack/RNSpokestackModule.java

```diff
@@ -86,6 +93,17 @@ public void initialize(ReadableMap config) throws Exception {
                     .setAndroidContext(reactContext.getApplicationContext())
                     .build();
         }
+
+        // NLU
+        if (config.hasKey("nlu")) {
+            final TensorflowNLU.Builder nluBuilder = new TensorflowNLU.Builder();
+            nluBuilder.setConfig(pipeline.getConfig());
+            nluBuilder.addTraceListener(this);
+            Map<String, Object> map = config.getMap("nlu").toHashMap();
```

Comment:
Where do the keys that need to be in here get documented?

Reply:
https://github.com/spokestack/react-native-spokestack#javascript

Reply:
also i misread this at first in a helpful way that pointed to missing the docs for the structure of the `nluEvent`, so that's nice.

Reply:
I don't see an `nlu` key in there; just the specific model, metadata, and vocab paths.

Reply:
ah you're correct, had to update that for android and didn't add it to the readme